src/libraries/System.Text.RegularExpressions/src/System/Text/RegularExpressions/Symbolic/SymbolicRegexNode.cs

```diff
@@ -1974,12 +1940,8 @@ internal bool StartsWithLoop(int upperBoundLowestValue = 1)
             };
         }
 
-
-        /// <summary>Gets the set that includes all elements that can start a match.</summary>
-        internal TSet GetStartSet() => _startSet;
-
         /// <summary>Computes the set that includes all elements that can start a match.</summary>
-        private TSet ComputeStartSet()
+        public TSet GetStartSet(SymbolicRegexBuilder<TSet> builder)
```

Comment:
What is the purpose of the start set now that we use the shared find optimizations for finding the next possible match location?  Is this necessary for certain anchors?

Reply:
Start set is still needed for PruneAnchors to figure out which anchors overlap with the possible next characters accepted by the pattern, but that's the only usage.